### PR TITLE
APPSERV-148 Release of MC 1.1

### PR DESCRIPTION
**Pull request by jbee**
Merge and run after #9 is merged.

### CODE REVIEW

File: process/pom.xml [hunk 1]
```diff
@@ -95,7 +95,7 @@
         <dependency>
             <groupId>fish.payara.monitoring-console</groupId>
             <artifactId>monitoring-console-api</artifactId>
-            <version>1.1-SNAPSHOT</version>
+            <version>1.0</version>
```
Comment (jbee) [inline process/pom.xml:98]: API version is stable 1.0 and should not change.

**Review by Pandrex247 — APPROVED** [review]

**Comment by pzygielo** [comment]
Could the release version, 428cb2c, be tagged, please?

**Comment by jbee** [comment]
@pzygielo yes, we do that as part of the release. The idea of the release PRs will be that they always have two commits where the release is the first one and the seconds incs version to snapshot again after release. We merge them and then tag and make a release.

**Comment by pzygielo** [comment]
@jbee  thanks for checking. Asking because I can't see tag for previous release 1.0 in this repo.

**Pull request merged**